list.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body class="body">
+  <div class="books-container row">
+    <div class="book column shadow-container">
+      <div class="book-info">
+        <img src="./assets/book-cover.png" alt="Book cover" class="book-cover-list">
+        <h4 class="book-list-title text-3">Título del libro</h4>
```

Comment:
It'd do `book-info-title`

Since this is not the title of the entire list, it's the title of a specific section. Info section to be specific. I think is more accurate. Semantic means a big deal when it comes to this things. Good code it's the one we can make clearer to understand for others